<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetError</class>
 <widget class="QWidget" name="WidgetError">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_4" native="true">
     <property name="styleSheet">
      <string notr="true">#widget_4{background-color: rgba(255, 255, 255, 0);}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="36,16,496,24,32,20">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="95,12,817">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{color: rgb(217, 231, 255);
background-color: rgb(55, 68, 88);border:none;}
QPushButton::hover{color: rgb(217, 231, 255);
background-color: rgb(66, 82, 106);border:none;}
QPushButton::pressed{color: rgb(217, 231, 255);
background-color: rgb(66, 82, 106);border:none;}
</string>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
           <property name="icon">
            <iconset resource="../image.qrc">
             <normaloff>:/image/home_all/fanhui_icon.png</normaloff>:/image/home_all/fanhui_icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="styleSheet">
            <string notr="true">#widget_3{background-color: rgb(55, 68, 88);}
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="109,378,93,14,18,10,93,21,54,16">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">QLabel{color: rgb(227, 186, 56);
background-color: rgba(255, 255, 255, 0);border:none;}
</string>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QDateEdit{color: rgb(57, 64, 75);
background-color: rgb(227, 186, 56);border:none;}
QDateEdit:hover{color: rgb(57, 64, 75);
background-color: rgb(227, 186, 56);}
QDateEdit::drop-down{width: 15px;image:  url(:/image/home_all/chevron-states.png);background-color:rgb(227, 186, 56);}
QCalendarWidget QWidget#qt_calendar_navigationbar{
                    background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,stop: 0 #4f596b, stop: 1 #4f596b);}
QCalendarWidget QToolButton{color:rgb(255, 226, 96);background-color: rgb(79, 89, 107);}
QCalendarWidget QToolButton#qt_calendar_prevmonth{qproperty-icon: url(:/image/home_all/left_icon.png);}
QCalendarWidget QToolButton#qt_calendar_nextmonth{qproperty-icon: url(:/image/home_all/right_icon.png);}</string>
              </property>
              <property name="dateTime">
               <datetime>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
                <year>2007</year>
                <month>1</month>
                <day>1</day>
               </datetime>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">border-image: url(:/image/home_all/link2.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QDateEdit{color: rgb(57, 64, 75);
background-color: rgb(227, 186, 56);border:none;}
QDateEdit:hover{color: rgb(57, 64, 75);
background-color: rgb(227, 186, 56);}
QDateEdit::drop-down{width: 15px;image:  url(:/image/home_all/chevron-states.png);background-color:rgb(227, 186, 56);}
QCalendarWidget QWidget#qt_calendar_navigationbar{
                    background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,stop: 0 #4f596b, stop: 1 #4f596b);}
QCalendarWidget QToolButton{color:rgb(255, 226, 96);background-color: rgb(79, 89, 107);}
QCalendarWidget QToolButton#qt_calendar_prevmonth{qproperty-icon: url(:/image/home_all/left_icon.png);}
QCalendarWidget QToolButton#qt_calendar_nextmonth{qproperty-icon: url(:/image/home_all/right_icon.png);}
</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="styleSheet">
               <string notr="true">QPushButton{color: rgb(227, 186, 56);
background-color: rgba(255, 255, 255, 0);border:none;}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../image.qrc">
                <normaloff>:/image/home_all/search_icon.png</normaloff>:/image/home_all/search_icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
